examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -1,8 +1,8 @@
 enabled = true
 
-region = "us-east-2"
+region = "us-west-2"
```

Comment:
@maximmi Should we rename this file? And I'm guessing we'd need to update the automated tests as well if we do.